pages/human-donors/[uuid].js

```diff
@@ -91,6 +94,14 @@ export default function HumanDonor({
           </>
         )}
         <Attribution award={award} lab={lab} />
+        {donor.collections?.length > 0 && (
+          <DataPanel>
+            <DataArea>
+              <DataItemLabel>Collections</DataItemLabel>
+              <DataItemValue>{donor.collections.join(", ")}</DataItemValue>
+            </DataArea>
+          </DataPanel>
+        )}
```

Comment:
You should add `collections` to the `<Attribution>` component instead of below it so that you don’t have the horizontal line between them. At some point we’ll probably need to have the `<Attribution>` component take children so that different pages can add their own parts to the Attribution panel, but we don’t yet know what kinds of things will be in this panel, so we can’t do that yet. _Many_ objects have a `collections` property, so it makes sense to make that a part of `<Attribution>` just like award and lab already are.